master/docs/relnotes/index.rst

```diff
@@ -21,6 +21,7 @@ Features
 ~~~~~~~~
 
 * :class:`GitPoller` now has a ``buildPushesWithNoCommits`` option to allow the rebuild of already known commits on new branches.
+* Add GitLab authentication plugin for web UI. See :class:`buildbot.www.oauth2.GitLabAuth`.
```

Comment:
@tardyp I believe this should go into 0.9.0b8.rst release notes.

I created template (and filled it with my stuff) in PR #2012.